adafruit_epd/epd.py

```diff
@@ -149,15 +165,15 @@ def display(self):  # pylint: disable=too-many-branches
 
         self.update()
 
-    def hardware_reset(self):
+    def hardware_reset(self) -> None:
         """If we have a reset pin, do a hardware reset by toggling it"""
         if self._rst:
             self._rst.value = False
             time.sleep(0.1)
             self._rst.value = True
             time.sleep(0.1)
 
-    def command(self, cmd, data=None, end=True):
+    def command(self, cmd: Any, data: Any = None, end: bool = True) -> Any:
```

Comment:
I think `cmd` is expected to be `int`. 

`data` can be `Optional[bytearray]` it needs `Optional` since it's allowed to be `None`.

And the return can be `int`. 

I found these types by adding some additional print statements into this function to check the types while executing the simpletest.


Once these changes are made here, it will also result in some of the other files that extend the class here to be updated in some places that currently are typed as `Any` where they are calling this `command()` function.